index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title></title>
+    <link rel="stylesheet" href="style.css">

```

Comment:
This is our custom stylesheet. If you remember from class, our custom stylesheet must go after all the other `link rel=` stylesheets. This is because we want our styling to be the last one applied to our HTML, rather than having our styling be overridden by presets from Tailwind, etc.

Could you please move this to the line after the Tailwind `script` tag?